source/ncengine/graphics2/NcGraphicsImpl2.cpp

```diff
@@ -193,19 +193,35 @@ NcGraphicsImpl2::NcGraphicsImpl2(const config::GraphicsSettings& graphicsSetting
                     .colorSink = MainColor,
                     .depthSink = MainDepth
                 },
+                PassDesc{
+                    .id = MaterialPassFlag::Toon,
+                    .name = "ToonSkinned",
+                    .type = PassType::SkinnedMaterial,
+                    .shaderPaths = ShaderPaths{"Toon.psh", "ToonSkinned.vsh"},
+                    .colorSink = MainColor,
+                    .depthSink = MainDepth
+                },
                 PassDesc{
                     .id = MaterialPassFlag::Normals,
                     .name = "Normals",
                     .type = PassType::Material,
                     .shaderPaths = ShaderPaths{"Normals.psh", "Toon.vsh"},
                     .colorSink = NormalsColor
                 },
+                PassDesc{
+                    .id = MaterialPassFlag::Normals,
+                    .name = "NormalsSkinned",
+                    .type = PassType::SkinnedMaterial,
+                    .shaderPaths = ShaderPaths{"Normals.psh", "ToonSkinned.vsh"},
+                    .colorSink = NormalsColor
+                },
                 PassDesc{
                     .id = MiscPassFlag::Wireframe,
                     .name = "Wireframe",
                     .type = PassType::Wireframe,
                     .shaderPaths = ShaderPaths{"Wireframe.psh", "Wireframe.vsh"},
-                    .colorSink = MainColor
+                    .colorSink = MainColor,
+                    .depthSink = MainDepth
```

Comment:
Will you double check this is legit? Wireframes were rendering on top of standard meshes, so I figured this was the ticket, but IDK.